packages/amplify-category-geo/src/provider-controllers/import.ts

```diff
@@ -0,0 +1,143 @@
+import { $TSContext } from 'amplify-cli-core';
+import { prompter, printer } from 'amplify-prompts';
+import { existsSync, writeFileSync } from 'fs-extra';
+import { join } from 'path';
+import aws from 'aws-sdk';
+import { ServiceName } from '../service-utils/constants';
+import { validateGeoJSONObj } from '../service-utils/validateGeoJSONObj';
+import {
+  FeatureCollection, ImportParams, GeofenceCollectionParams, GeofenceParams, IdentifierOption, IdentifierInfo,
+} from '../service-utils/importParams';
+import { getGeoResources, getGeoServiceMeta } from '../service-utils/resourceUtils';
+
+const MAX_ENTRIES_PER_BATCH = 10;
+
+export const importResource = async (context: $TSContext) => {
+  const geofenceCollectionResourcesMap = await getGeoServiceMeta(ServiceName.GeofenceCollection);
+  const collectionNames = await getGeoResources(ServiceName.GeofenceCollection);
+  if (collectionNames.length === 0) {
+    throw new Error('Geofence collection is not found. Run `amplify geo add` to create a new geofence collection.')
+  }
+  // Get provisioned geofence collection name
+  const provisionedCollectionNames = collectionNames.filter(collectionName => {
+    const collection = geofenceCollectionResourcesMap[collectionName];
+    if (!collection.output || !collection.output.Name || !collection.output.Region) {
+      return false;
+    }
+    return true;
+  });
+  if (provisionedCollectionNames.length === 0) {
+    throw new Error('No geofence is not provisioned yet. Run \`amplify push\` to provision geofence collection.')
+  }
+  // Get collection region
+  const collectionRegion = geofenceCollectionResourcesMap[provisionedCollectionNames[0]].output.Region;
+  // Get collection output name
+  const provisionedCollectionOutputNames = provisionedCollectionNames.map(collectionName => geofenceCollectionResourcesMap[collectionName].output.Name);
+  // Get the collection to import
+  let collectionToImport: string = provisionedCollectionOutputNames[0];
+  if (provisionedCollectionOutputNames.length > 1) {
+    if (provisionedCollectionOutputNames.length < collectionNames.length) {
+      printer.warn('There are additional geofence collections in the project that have not been deployed. To import data into these resources, run `amplify push` first.');
+    }
+    collectionToImport = await prompter.pick<'one', string>('Select the Geofence Collection to import with Geofences', provisionedCollectionOutputNames);
+  }
+  // Ask for geo json file path
+  let geoJSONFilePath: string;
+  geoJSONFilePath = join(await prompter.input(`Provide the path to GeoJSON file containing the Geofences for ${collectionToImport} collection. Refer https://geojson.io/ for a sample GeoJSON:`));
+  while (!existsSync(geoJSONFilePath)) {
+    geoJSONFilePath = join(await prompter.input(`Cannot find GeoJSON file. Re-enter a valid file path:`));
+  }
+  let geoJSONObj: FeatureCollection;
+  geoJSONObj = JSON.parse(readFileSync(geoJSONFilePath, 'utf-8')) as FeatureCollection;
```

Comment:
use JSONUtilities from `amplify-cli-core`:
```suggestion
  geoJSONObj = JSONUtilities.readJson<FeatureCollection>(geoJSONFilePath);
```